examples/structured_outputs/structured-outputs.ts

```diff
@@ -0,0 +1,50 @@
+import { Ollama } from '../../src/index.js';
+import { z } from 'zod';
+import { zodToJsonSchema } from 'zod-to-json-schema';
+
+const ollama = new Ollama();
+
+// Define the schema for friend info
+const FriendInfoSchema = z.object({
+    name: z.string(),
+    age: z.number().int(),
+    is_available: z.boolean()
+});
+
+// Define the schema for friend list
+const FriendListSchema = z.object({
+    friends: z.array(FriendInfoSchema)
+});
+
+async function run() {
+    // Convert the Zod schema to JSON Schema format
+    const jsonSchema = zodToJsonSchema(FriendListSchema);
+
+    // Can use manually defined schema directly
```

Comment:
This would be more easily readable as a multi-line comment to me. 

Ex:
```
/*
Can use manually defined schema directly:
{
  'type': 'object',
  'properties': {
    'friends': {
      'type': 'array',
      'items': {
        'type': 'object',
        'properties': {
          'name': { 'type': 'string' },
          'age': { 'type': 'integer' },
          'is_available': { 'type': 'boolean' }
        },
        'required': ['name', 'age', 'is_available']
      }
    }
  },
  'required': ['friends']
}
*/
```